packages/picasso.js/src/core/chart-components/axis/axis.js

```diff
@@ -22,10 +22,13 @@ function alignTransform({ align, inner }) {
 function resolveAlign(align, dock) {
   const horizontal = ['top', 'bottom'];
   const vertical = ['left', 'right'];
-  if (horizontal.indexOf(align) !== -1 && vertical.indexOf(dock) === -1) {
-    return align;
-  }
-  if (vertical.indexOf(align) !== -1 && horizontal.indexOf(dock) === -1) {
+
+  const docksHorizontally = horizontal.includes(dock);
```

Comment:
Tests on IE11 are failing most likely because of this: we don't have a polyfill for `includes`

Reply:
Ok, I should probably revert that whole thing actually.